--- libsupp/clif.c	2007-08-06 17:20:17.000000000 +0200
+++ libsupp/clif.c.oden	2008-12-25 22:14:43.000000000 +0100
@@ -792,9 +792,9 @@ static void box_output (int start, int l
 	if (l > 0) {
 	    memset (buf, ' ', l);
 	    buf[l] = '\0';
-	    fprintf (stderr, buf);
+	    fprintf (stderr, "%s", buf);
 	} else 
-	    fprintf (stderr, spacer);
+	    fprintf (stderr, "%s", spacer);
 
 
 	endp = buf + width;
@@ -833,7 +833,7 @@ static void box_output (int start, int l
 		    
 	    *p = '\0';
 	    fprintf (stderr, "%s", buf);
-	    fprintf (stderr, spacer);
+	    fprintf (stderr, "%s", spacer);
 		
 	    p++;
 	    for (s = buf; *p; *s++ = *p++) ;
